mapper/pojo-base/src/main/java/org/hibernate/search/mapper/pojo/mapping/building/impl/PojoMapper.java

```diff
@@ -93,19 +96,21 @@ public PojoMapper(MappingBuildContext buildContext,
 			TypeMetadataContributorProvider<PojoTypeMetadataContributor> contributorProvider,
 			PojoBootstrapIntrospector introspector,
 			ContainerExtractorRegistry containerExtractorRegistry,
-			boolean implicitProvidedId,
+			BeanReference<? extends IdentifierBridge<Object>> identifierBridge,
```

Comment:
Same here, I would rename this parameter to `providedIdentifierBridgeReference`.